### PR TITLE
feat(1167): [5][small] Use wrapper script

### PR DESCRIPTION
**Pull request by kumada626**
## Context

To fetch build token in actual build execution, we fixed to exchange temporal token to build token in launcher process. 
However, also log-service uses token which is passed by `executor-*`.
To pass fetched build token to log-service and launcher, add the flag to launcher which only fetch token in launcher process, and use this flag to set fetched token before launcher and log-service processes start.
Detail: https://github.com/screwdriver-cd/screwdriver/issues/1167#issuecomment-409441739

Launcher passes wrapper script to run build. Therefore, each executor uses this script.

## Objective

Use wrapper script to run build.

## References

issue: https://github.com/screwdriver-cd/screwdriver/issues/1167
issue comment: https://github.com/screwdriver-cd/screwdriver/issues/1167#issuecomment-409441739

PR:
launcher: https://github.com/screwdriver-cd/launcher/pull/203
executor-k8s: https://github.com/screwdriver-cd/executor-k8s/pull/89
executor-docker: https://github.com/screwdriver-cd/executor-docker/pull/28
executor-jenkins: https://github.com/screwdriver-cd/executor-jenkins/pull/24

### CODE REVIEW

**Review by tk3fftk — APPROVED** [review]

**Review by wdstar — APPROVED** [review]

**Review by jithine — COMMENTED** [review]
same feed back as in other PR's we should remove SD_TOKEN from envs. lines 32-35

File: scripts/hyper-pod-template.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
         "--",
         "/bin/sh",
         "-c",
-        "while ! [ -f /opt/sd/setup.sh ]; do sleep 1; done; /opt/sd/setup.sh && (/opt/sd/launch --api-uri API_URI --store-uri STORE_URI --emitter /sd/emitter --build-timeout SD_BUILD_TIMEOUT BUILD_ID & /opt/sd/logservice --emitter /sd/emitter --api-uri API_URI --store-uri STORE_URI --build BUILD_ID & wait $(jobs -p))"
+        "while ! [ -f /opt/sd/setup.sh ]; do sleep 1; done; /opt/sd/setup.sh && (SD_TOKEN=`/opt/sd/launch --only-fetch-token --token \"BUILD_TOKEN\" --api-uri API_URI --store-uri STORE_URI --emitter /sd/emitter --build-timeout SD_BUILD_TIMEOUT BUILD_ID` && (/opt/sd/launch --api-uri API_URI --store-uri STORE_URI --emitter /sd/emitter --build-timeout SD_BUILD_TIMEOUT BUILD_ID & /opt/sd/logservice --emitter /sd/emitter --api-uri API_URI --store-uri STORE_URI --build BUILD_ID & wait $(jobs -p)))"
```
Comment (jithine) [inline scripts/hyper-pod-template.json]: I am wondering should we just provide this in a wrapper script. The whole 
```
(SD_TOKEN=`/opt/sd/launch --only-fetch-token --token \"BUILD_TOKEN\" --api-uri API_URI --store-uri STORE_URI --emitter /sd/emitter --build-timeout SD_BUILD_TIMEOUT BUILD_ID` && (/opt/sd/launch --api-uri API_URI --store-uri STORE_URI --emitter /sd/emitter --build-timeout SD_BUILD_TIMEOUT BUILD_ID & /opt/sd/logservice --emitter /sd/emitter --api-uri API_URI --store-uri STORE_URI --build BUILD_ID & wait $(jobs -p))
``` 

part in `/opt/sd/run.sh`

I am seeing that we are repeating this in all executors, and can be error prone. So why not wrap this in a script which launcher exposes and controls ? So all executors have to do is invoke `/opt/sd/run.sh` in addition to any extra scripts they have to run

Comment (jithine) [reply scripts/hyper-pod-template.json]: Input to the script `/opt/sd/run.sh` will be TEMP_TOKEN, API_URI,  STORE_URI, SD_BUILD_TIMEOUT & BUILD_ID

I'm just worried that we are putting logic of fetching token in multiple executors. A better approach might be to let launcher handle it along with invocation of logservice.

Comment (kumada626) [reply scripts/hyper-pod-template.json]: I think it should better to provide a wrapper script. I'll fix launcher and executors to use such wrapper script for the first step. To fix to let launcher handle logic of fetching token along with invocation of logservice is a second step.

**Review by jithine — APPROVED** [review]

**Review by jithine — APPROVED** [review]
why not double quote all variables ? But I guess since none of them are user controlled we should be okay.

**Comment by kumada626** [comment]
There is no specific reason not to double quote all variables. I fixed to double quote all variables.

**Review by jithine — APPROVED** [review]

**Review by tk3fftk — APPROVED** [review]

**Pull request merged**